testsuites/syncgateway/functional/tests/test_no_conflicts.py

```diff
@@ -673,7 +679,7 @@ def test_revs_cache_size(params_from_base_test_setup, sg_conf_name, num_of_docs)
     sync_gateway_version = params_from_base_test_setup["sync_gateway_version"]
 
     if sync_gateway_version < "2.0":
-        pytest.skip('It does not work with sg < 2.0 , so skipping the test')
+        pytest.skip('--no-conflicts is enabled and does not work with sg < 2.0 , so skipping the test')
```

Comment:
can you remove this text '--no-conflicts is enabled' as condition is only for sg version

Reply:
done